Tests/Pcap++Test/TestDefinition.h

```diff
@@ -64,6 +64,7 @@ PTF_TEST_CASE(TestTcpReassemblySanity);
 PTF_TEST_CASE(TestTcpReassemblyRetran);
 PTF_TEST_CASE(TestTcpReassemblyMissingData);
 PTF_TEST_CASE(TestTcpReassemblyOutOfOrder);
+PTF_TEST_CASE(TestTcpReassemblyOOOWithManualClose);
```

Comment:
what is "OOO"?

Reply:
OOO == Out Of Order

Reply:
I don't like the abbreviation, not straightforward, I would recommand `TestTcpReassemblyOutOfOrderWithManualClose`

Reply:
The problem is that we currently have a limit of 35 chars per test (you can see it everywhere in `PcppTestFramework.h` so longer test names will look misaligned when running them. Perhaps we can remove this limitation and automatically detect the longest test name, but I'm not sure it'd be straightforward to do 🤔 

We can also extend the 35 limit to a larger number...

Reply:
@seladb  maybe we should increase the number? `TestTcpReassemblyOutOfOrderWithManualClose` length is 42, so I guess changing it to 50 is enough.